packages/xo-server-audit/src/index.js

```diff
@@ -20,38 +20,56 @@ const DEFAULT_BLOCKED_LIST = {
   'audit.checkIntegrity': true,
   'audit.generateFingerprint': true,
   'audit.getRecords': true,
+  'backup.list': true,
+  'backupNg.fetchFiles': true,
   'backupNg.getAllJobs': true,
   'backupNg.getAllLogs': true,
+  'backupNg.listFiles': true,
+  'backupNg.listPartitions': true,
+  'backupNg.listVmBackups': true,
   'cloud.getResourceCatalog': true,
   'cloudConfig.getAll': true,
   'group.getAll': true,
+  'host.isHostServerTimeConsistent': true,
+  'host.isHyperThreadingEnabled': true,
   'host.stats': true,
   'ipPool.getAll': true,
   'job.getAll': true,
   'log.get': true,
   'metadataBackup.getAllJobs': true,
+  'network.getBondModes': true,
+  'pif.getIpv4ConfigurationModes': true,
   'plugin.get': true,
+  'pool.getPatchesDifference': true,
   'pool.listMissingPatches': true,
   'proxy.getAll': true,
   'remote.getAll': true,
   'remote.getAllInfo': true,
+  'remote.list': true,
+  'remote.test': true,
   'resourceSet.getAll': true,
   'role.getAll': true,
   'schedule.getAll': true,
   'server.getAll': true,
   'session.getUser': true,
   'session.signIn': true,
   'sr.getUnhealthyVdiChainsLength': true,
+  'sr.scan': true,
```

Comment:
Why?

Reply:
`sr.scan` refresh the SR VDIs. It doesn't have any impact on the infrastructure.

Reply:
It does, it refreshes the internal state of the SR, looking for existing VDIs.

Also, it's not an automatic action, it's a manual one, I don't see the point of ignoring it.

Reply:
Done :+1: